frontend/playground-frontend/src/components/AccountSettings/AccountSettings.tsx

```diff
@@ -21,13 +18,17 @@ const SettingsBlock = () => {
       await updateUserSettings(fullName, email, password);
     }
   };
+
+  if (!user) {
+    return <></>;
```

Comment:
there is a bit of a cleaner way to do this. Have a helper function that renders the account settings form and do something like 

```
return (
{if user && renderAccountSettingsForm()}
)
```